docs/architecture/decisions/0006-unique-ids-for-private-beta.md

```diff
@@ -41,24 +41,28 @@ Thus a valid LPA number would be M-0000-0000-0013 where the 3 is a checkdigit ge
 
-- LPA reference numbers that are purely numeric are more guessable with enough tries or given sufficinet requests. They should not be used as publically facing identifiers in web based services without some 2nd factor (for example the paper verification codes) and any services using them as url identifiers or form parameters should put in place detection for brute forcing and iteration based attacks.
+- LPA reference numbers that are purely numeric are more guessable with enough tries or given sufficient requests. They should not be used as publicly facing identifiers in web based services without some 2nd factor (for example the paper verification codes) and any services using them as url identifiers or form parameters should put in place detection for brute forcing and iteration based attacks.
 
-- They should not be used in URLs on external-facing systems.
+- They should not be used in URLs on external-facing systems. In external facing services or APIs that need to make direct reference to an LPA the UUID associated with an LPA should be used instead.
 
-- During private beta it is essential we track LPA ref number usage and adjust our approach accordingly. Particularly if more drafts are created than expected as this could use up the available set of IDs at a greater rate. A checkpoint at the end of private beta to assess this should be put in place.
+- During private beta it is essential we track LPA reference number usage and adjust our approach accordingly. Particularly if more drafts are created than expected as this could use up the available set of IDs at a greater rate. A checkpoint at the end of private beta to assess this should be put in place.
 
 - The reduced information space will mean we may have to use a different technology in the UID generator due to likelihood of collisions from random generation, potentially a traditional RDS with a sequence generator. This will create rework for teams.
 
-- If ID space looks like it is close to exhausted, OPG has the option to implement a successor prefix i.e. M-0000 0000 0001 to L-0000 0000 0001. This means the prefix is part of the number and should asked for and stored in all interactions.
+- If ID space looks like it is close to exhausted, OPG has the option to implement a successor prefix i.e. M-0000 0000 0001 to L-0000 0000 0001.
 
-- Modernised LPA reference numbers should be used only for LPAs applications, not sub actors such as attorneys or non-lpa cases like supervision orders to reduce usage.
+- The prefix is part of the LPA reference number and should asked for and stored in all interactions cases. The reference number should not be stored in a database without the prefix, so should be stored as a string. Formatting, such as dashes or spaces to improve readability are not part of the data, so are not stored.
```

Comment:
We currently do store dashes as part of the reference number and, from experience with Sirius, it's a lot easier to do so because otherwise you're constantly reformatting it. e.g. Sirius stores legacy reference numbers like 700212345678 but converts them into 7002-1234-5678 on every database fetch because that's the only way they're ever displayed or used.

Reply:
Given we're already a way down that path I'll remove that note then.